build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -275,19 +279,79 @@ download_all()
 		echo "Downloading ${parts[$partidx]} binary"
 		echo "=========================="
 
-		./amebad_image_tool $TTYDEV 1 ${offsets[$partidx]} ${exe_name}
-		rm ${exe_name}
-
+		./amebad_image_tool 1 $TTYDEV 1 ${offsets[$partidx]} ${exe_name}
 	done
 	echo ""
 	echo "Download COMPLETE!"
 }
 
+erase()
+{
+	cd ${IMG_TOOL_PATH}
+	echo "Starting Erase..."
+	ota_addr=$(($FLASH_START_ADDR + $CONFIG_AMEBAD_FLASH_CAPACITY))
+	if [[ $1 == "erase_all" || $1 == "ERASE_ALL" ]];then
```

Comment:
Could you separate `erase` and `partition` like `./dbuild download erase all` to make it the same as other boards.

Reply:
ok, I am changing on this